docs/modules/getting-started/pages/get-started-docker.adoc

```diff
@@ -313,28 +313,60 @@ go get github.com/hazelcast/hazelcast-go-client
 +
 [source,go]
 ----
-import "github.com/hazelcast/hazelcast-go-client"
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+	ctx := context.TODO()
 
-func mapSampleRun() {
-  cb := hazelcast.NewConfigBuilder()
+	cb := hazelcast.NewConfig()
+	cb.ClientName = "hello-world" <1>
 
-  cb.Cluster().SetName("hello-world") <1>
+	hz, err := hazelcast.StartNewClientWithConfig(ctx, cb) <2>
+	if err != nil {
+		log.Fatalf("Client could not connect to the cluster, %v", err)
+	}
 
-  client, err := hazelcast.StartNewClientWithConfig(cb) <2>
+	defer func(ctx context.Context) { <5>
```

Comment:
I think we can use the context from the outer scope. We do not need to pass it with this parameter. Wdyt?

Reply:
If I would not send as a parameter it will take main scope's ctx value as a reference I guess. We are also sending context as a parameter as a convention. Maybe, there are something which need to be copied while sending as a parameter.  That's why I sent context in this way. It can be bad practice. I'm not sure.